src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function getCalendarWidth($size) {
+  @return ($size * 7) + ($day-block-gap * 6) + ($field-padding * 2);
+}
```

Comment:
You can modify your function to reuse it calculating margin-left for start-day modifiers

Reply:
As for me calculating margin-left better to do inline,  because i can easy to understand how it is calculated